### PR TITLE
Modify openstack provider configuration for CCEE

### PR DESCRIPTION
**Pull request by voigt**
- To expose the APIServer over the public internet, We have to modify Exposure Class in the shoot spec
- FloatingIP Network for CCEE is `FloatingIP-external-cp-kyma`

Based on this PR [#898](https://github.com/kyma-project/control-plane/pull/898)
Issue: [#1775](https://github.tools.sap/kyma/backlog/issues/1775)

TODO:

- [x] implementation

### CODE REVIEW

File: components/kyma-environment-broker/internal/provider/openstack_provider.go [hunk 1]
```diff
@@ -4,33 +4,39 @@ import (
...
 
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/ptr"
+
 	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
 	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
 )
 
 const (
 	DefaultOpenStackRegion = "eu-de-1"
+	// DefaultKymaExternalFloatingIPNetwork = "FloatingIP-external-cp-kyma"
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/provider/openstack_provider.go:15]: do we need this?

File: components/kyma-environment-broker/internal/process/input/builder.go [hunk 2]
```diff
@@ -100,6 +100,7 @@ func (f *InputBuilderFactory) IsPlanSupport(planID string) bool {
 	}
 }
 
+// inputs are created here
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/process/input/builder.go:103]: please remove the comment

**Review by crabtree — APPROVED** [review]

**Pull request merged**